src/field_theory/adjoin.lean

```diff
@@ -618,6 +618,22 @@ begin
     exact ih (adjoin F s) a h1 h4 }
 end
 
+lemma adjoin.finite_dimensional_of_finite_set {K L : Type*} [field K] [field L] [algebra K L]
+  {S : set L} (hS : S.finite) (h_int : ∀ x ∈ S, is_integral K x) :
+finite_dimensional K (intermediate_field.adjoin K S) :=
```

Comment:
Could you also fix the indentation on the new stuff in krull_topology? (see https://leanprover-community.github.io/contribute/style.html#structuring-definitions-and-theorems)
```suggestion
  finite_dimensional K (intermediate_field.adjoin K S) :=
```

Reply:
I think I fixed it. Please let me know if there's still something wrong!

Reply:
Looks a lot better. Thanks!

Reply:
It looks like this got reverted

Reply:
@Sebastian-Monnet this comment still applies